ru/turing/courses/lesson2/animals/Cat.java

```diff
@@ -0,0 +1,35 @@
+package ru.turing.courses.lesson2.animals;
+
+public class Cat extends Animal {
+    private boolean wanna_play; // 1 - want to ... 0 - don't want to
+    private int mice_count = 0;
```

Comment:
```suggestion
    private int miceCount = 0;
```
В джаве используется CamelCase